dashboard/README.md

```diff
@@ -0,0 +1,48 @@
+# Material UI - Next.js Pages Router example in TypeScript
+
+## How to use
```

Comment:
Could you update the readme so that it reflects the context from this repo? E.g. you do not need to download the example anymore